target/linux/mxs/config-6.6

```diff
@@ -86,13 +88,14 @@ CONFIG_EXTCON=y
 CONFIG_FEC=y
 CONFIG_FIXED_PHY=y
 CONFIG_FIX_EARLYCON_MEM=y
+CONFIG_FORCE_NR_CPUS=y
 CONFIG_FS_IOMAP=y
 CONFIG_FS_MBCACHE=y
 CONFIG_FUNCTION_ALIGNMENT=0
 CONFIG_FWNODE_MDIO=y
 CONFIG_FW_LOADER_PAGED_BUF=y
 CONFIG_FW_LOADER_SYSFS=y
-CONFIG_GCC11_NO_ARRAY_BOUNDS=y
+CONFIG_GCC10_NO_ARRAY_BOUNDS=y
```

Comment:
mhhh really sounds like a good candiate for the filter...

Reply:
Yep. https://github.com/openwrt/openwrt/pull/16770

Reply:
removed.